ARExperiment/ARExperiment/LightsAnimationsViewController.swift

```diff
@@ -42,12 +40,18 @@ class LightsAnimationsViewController: UIViewController {
         sceneView.session.pause()
     }
 
-    private func createSceneNodeForAsset(_ assetName: String, assetPath: String) -> SCNNode? {
-        guard let paperPlaneScene = SCNScene(named: assetPath) else {
-            return nil
-        }
-        let carNode = paperPlaneScene.rootNode.childNode(withName: assetName, recursively: true)
-        return carNode
+    private func setUpModelsOnLoad() {
+        objectNodeModel = modelForNodeName(objectNode1)
```

Comment:
nice abstraction here! but perhaps lets rename `objectNode`, as this is always loaded with a sphere, then we can be more specific and call it `sphereNode`, or potentially something even clearer

Reply:
I mean is always sphere now, I did not want to make the controllers objects dependent on which model they were showing.

Reply:
well, let's be specific for now, don't optimise for something that might not happen, at the expense of clarify!